app/controllers/spree/paypal_controller.rb

```diff
@@ -0,0 +1,247 @@
+# frozen_string_literal: true
+
+module Spree
+  class PaypalController < StoreController
```

Comment:
Thank you, I have closed this PR now: https://github.com/openfoodfoundation/better_spree_paypal_express/pull/11/files